Mathlib/Data/Finset/Card.lean

```diff
@@ -130,11 +130,21 @@ theorem card_doubleton (h : a ≠ b) : ({a, b} : Finset α).card = 2 := by
   rw [card_insert_of_not_mem (not_mem_singleton.2 h), card_singleton]
 #align finset.card_doubleton Finset.card_doubleton
 
+/-- `#(s \ {a}) = #s - 1` if `a ∈ s`. -/
```

Comment:
I'm not a huge fan of using `` `code` `` blocks to display stuff that isn't lean code, especially here where it *is* lean code but means something different! Perhaps better to write it as $\LaTeX$?